client/apps.go

```diff
@@ -70,3 +76,66 @@ func (client *Client) Apps(names []string, opts AppOptions) ([]*AppInfo, error)
 
 	return appInfos, err
 }
+
+// LogOptions represent the options of the Logs call.
+type LogOptions struct {
+	N      int  // The maximum number of log lines to retrieve initially. If <0, no limit.
+	Follow bool // Whether to continue returning new lines as they appear
+}
+
+// A Log holds the information of a single syslog entry
+type Log struct {
+	Timestamp time.Time `json:"timestamp"` // Timestamp of the event, in RFC3339 format to µs precision.
+	Message   string    `json:"message"`   // The log message itself
+	SID       string    `json:"sid"`       // The syslog identifier
+	PID       string    `json:"pid"`       // The process identifier
+}
+
+func (l Log) String() string {
+	return fmt.Sprintf("%s %s[%s]: %s", l.Timestamp.Format(time.RFC3339), l.SID, l.PID, l.Message)
+}
+
+// Logs asks for the logs of a series of services, by name.
+func (client *Client) Logs(names []string, opts LogOptions) (<-chan Log, error) {
+	query := url.Values{}
+	if len(names) > 0 {
+		query.Set("names", strings.Join(names, ","))
+	}
+	query.Set("n", strconv.Itoa(opts.N))
+	if opts.Follow {
+		query.Set("follow", strconv.FormatBool(opts.Follow))
+	}
+
+	rsp, err := client.raw("GET", "/v2/logs", query, nil, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	ch := make(chan Log, 20)
+	go func() {
```

Comment:
can you (re)describe the format expected here?  just reading the code I'm confused, maybe is clearer again later